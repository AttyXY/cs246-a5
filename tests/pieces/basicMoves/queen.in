setup
+ K c4
+ k f5 
+ Q a1
done
game human human
move a1 a8
move f5 f4
move a8 h8
move f4 f5
move h8 h1
move f5 f4
move h1 a1
move f4 f5
move a1 h8
move f5 e6
move h8 a1
move e6 f5
move a1 a8
move f5 e6
move a8 h1
move e6 f5
move h1 a8
quit
quit
