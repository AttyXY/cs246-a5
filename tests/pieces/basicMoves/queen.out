WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

WELCOME TO SETUP MODE!
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- + [PieceType] [Coordinate] (to add piece)
- - [Coordinate] (to remove piece)
- = [Colour] (to set turn)
- done (to exit setup mode)

8  --------
7  --------
6  --------
5  --------
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  Q-------
   abcdefgh

PLEASE ENTER THE FOLLOWING COMMANDS TO BEGIN:
- game [player1] [player2]

STARTING GAME!

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  Q-------
   abcdefgh

8  Q-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  Q-------
7  --------
6  --------
5  --------
4  --K--k--
3  --------
2  --------
1  --------
   abcdefgh

8  -------Q
7  --------
6  --------
5  --------
4  --K--k--
3  --------
2  --------
1  --------
   abcdefgh

8  -------Q
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  -------Q
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --K--k--
3  --------
2  --------
1  -------Q
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --K--k--
3  --------
2  --------
1  Q-------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  Q-------
   abcdefgh

8  -------Q
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  -------Q
7  --------
6  ----k---
5  --------
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  ----k---
5  --------
4  --K-----
3  --------
2  --------
1  Q-------
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  Q-------
   abcdefgh

8  Q-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  Q-------
7  --------
6  ----k---
5  --------
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh

8  --------
7  --------
6  ----k---
5  --------
4  --K-----
3  --------
2  --------
1  -------Q
   abcdefgh

8  --------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  -------Q
   abcdefgh

8  Q-------
7  --------
6  --------
5  -----k--
4  --K-----
3  --------
2  --------
1  --------
   abcdefgh


PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0
Black: 0
